### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by gickowtf**

### CODE REVIEW

**Comment by gickowtf** [comment]
hope u enjoy

**Comment by Skipperro** [comment]
It's mostly good, but I won't merge it with my repository yet, because if I'll update bucket name from "my-new-bucket" to "enpal" everything will brake. I will have to wait until Enpal will update software on my installation. Hopefully it won't take long.

File: custom_components/enpal/config_flow.py [hunk 1]
```diff
@@ -16,9 +16,10 @@
 big_int = vol.All(vol.Coerce(int), vol.Range(min=300))
 
 _LOGGER = logging.getLogger(__name__)
+
 CONFIG_SCHEMA = vol.Schema(
             {
-                vol.Required('enpal_host_ip', default='192.168.178.'): cv.string,
+                vol.Required('enpal_host_ip', default='192.168.137.36'): cv.string,
```
Comment (Skipperro) [inline custom_components/enpal/config_flow.py:22]: Putting your own IP as default for everyone - I don't like this idea. 
If someone will forget to change it or won't know that it have to be changed, stuff will not work.

192.168.178. - that's not complete, so you know it is to be edited and it's most popular IP pattern in Germany.

File: custom_components/enpal/sensor.py [hunk 2]
```diff
@@ -16,18 +16,17 @@
 from influxdb_client import InfluxDBClient
 
 _LOGGER = logging.getLogger(__name__)
-SCAN_INTERVAL = timedelta(seconds=120)
+SCAN_INTERVAL = timedelta(seconds=20)
 
 VERSION= '0.1.0'
```
Comment (Skipperro) [inline custom_components/enpal/sensor.py:21]: You've updated manifest to version 0.2.0, so why not this line?

File: custom_components/enpal/config_flow.py (hunk 1 above)
Comment (gickowtf) [reply custom_components/enpal/config_flow.py:22]: yes that was my mistake -.-

File: custom_components/enpal/sensor.py (hunk 2 above)
Comment (gickowtf) [reply custom_components/enpal/sensor.py:21]: next mistake -.-

**Comment by gickowtf** [comment]
Nonsense, everything is ok. I have absolutely no problem with it. I can work with my public fork and those who already have the Solar Rel web interface too. I'm with you and cross my fingers that Enpal works quickly on your update. or you make bucket and org configurable....

File: custom_components/enpal/sensor.py (hunk 2 above)
Comment (gickowtf) [reply custom_components/enpal/sensor.py]: fixed it in this branch iam sorry

**Comment by Skipperro** [comment]
> Nonsense, everything is ok. I have absolutely no problem with it.

Yes, because you have new software version, but I don't. If I'll run your code on my installation it won't work - not only because of bucket name, but also the content of the bucket. For example - in my influxdb there is no data for battery and wallbox. My battery sensor was just estimatation.

**Comment by gickowtf** [comment]
crazy ... ok hope that Enpal works quickly on your update

**Comment by Skipperro** [comment]
Good news - I may get my update soon. Just received email (looks like auto generated one) from Enpal:

> Hallo liebe Enpal-Kundin, lieber Enpal-Kunde,
> gerne möchten wir eine kurze Wartungsarbeit an ihrer Enpal-Box durchführen. Hierbei müssen wir ein Teil austauschen, das möglicherweise anfällig für Störungen sein kann. Um dies zu verhindern, möchten wir vorbeugend ihre Enpal-Box warten.
> 
> Ein Partnerbetrieb wird sich in den nächsten Tagen bei Ihnen melden und einen Termin mit Ihnen vereinbaren. Die Dauer des Termins wird ca. 1 Stunde betragen.
> 
> Vielen Dank für Ihr Verständnis!
> Liebe Grüße,

**Comment by gickowtf** [comment]
That sounds really good. I had the first two installs of my fork today via a forum! So my fork seems to work as it should. I hope the company gets back to you very soon! Also, a user has contacted me via yaml config also calculates the pure solar yield etc.! Maybe we get this pseudo entities yes also still with in the programming

**Comment by OlwinFroon** [comment]
Don't hold your breath, judging from their glacial speed I wouldn't hope for a unified new version anytime soon.

Unless you're willing to ask the user for the version they have (or let the software find out), I guess the best call would be to maintain your two versions until further notice: Skip's for the grey box and Gicko's for the white box. 

It would be important to advertise them properly, though, so the users won't strand with the wrong version.

**Pull request merged**